src/EditorFeatures/Core/Interactive/InteractiveSession.cs

```diff
@@ -86,6 +91,7 @@ internal sealed class InteractiveSession : IDisposable
             _workingDirectory = initialWorkingDirectory;
 
             _hostDirectory = Path.Combine(Path.GetDirectoryName(typeof(InteractiveSession).Assembly.Location)!, "InteractiveHost");
+            CreateEmptyEditorConfigFile(_hostDirectory);
```

Comment:
>CreateEmptyEditorConfigFile [](http://example.com/codeflow?start=12&length=27)

Oh, this will definitely fail since the host directory is VS install dir. Need to include the empty file in the VSIX instead.

Reply:
Hmm based on permissions?  This ran fine for me and created it in the appdata extension directory, e.g.
```C:\Users\dabarbet\AppData\Local\Microsoft\VisualStudio\17.0_99434ad3RoslynDev\Extensions\Microsoft\Roslyn Language Services\42.42.42.4242424\InteractiveHost```

Reply:
Ah like this - 
https://github.com/dotnet/roslyn/blob/09c9db28fc79823152a73b654f6521dc4b6dfa16/src/Interactive/HostProcess/InteractiveHost64.csproj#L33

That does seem better so I'll do that anyway

Reply:
> Hmm based on permissions? This ran fine for me and created it in the appdata extension directory, e.g.

Yeah. In general it's better to not write into that dir.